plugin-server/src/utils/db/db.ts

```diff
@@ -646,15 +658,15 @@ export class DB {
     public async addDistinctId(person: Person, distinctId: string): Promise<void> {
         const kafkaMessage = await this.addDistinctIdPooled(person, distinctId)
         if (this.kafkaProducer && kafkaMessage) {
-            await this.kafkaProducer.queueMessage(kafkaMessage)
+            await this.kafkaProducer.queueMessage(kafkaMessage[0])
```

Comment:
nit: this function is a bit confusing to me, but seems it was there before. given your change though, let's call this `kafkaMessages`